Casks/lilypond-unofficial.rb

```diff
@@ -0,0 +1,45 @@
+cask 'lilypond-unofficial' do
```

Comment:
How about `lilypond-64-bit` or something like that?

Reply:
The file would also have to be renamed, and that would most likely involve `git mv OLD_FILE_NAME NEW_FILE_NAME`.

Reply:
Ok, cask renamed,  
What do you want more ?